### ISSUE TITLE
oddity found with miniupnpc download tarball

### ISSUE THREAD

**Issue by taw00** [dashpay/dash#3247]
Oddity: miniupnpc-2.0.20170509.tar.gz as downloaded by the depends system versus from the same URL by browser download (Firefox) produces a tarball with two different sha256sums:
    By browser:    `b0dbfeafb674b7e2dbb53b9ae071031c34258d94f6224ddb6c17cc20b0f9b3d2`
    By depends or wget: `d3c368627f5cdfb66d3ebd64ca39ba54d6ff14a61966dbecb8dd296b7039f16a`

The dashcore build expects the second.

Relevant path in source tree: `dash-0.15.0.0-rc1/depends/packages/miniupnpc.mk`  
The URL is the same: http://miniupnp.free.fr/files/miniupnpc-2.0.20170509.tar.gz

Am I crazy? This may or may not be relevant, but ... so odd. Going to try different browsers.

UPDATE: Download by wget (I haven't tried curl) and you get the tarball with the sha256sum you want. Download by Firefox or Chrome and you get the one that ends in 9b3d2

I don't know how important this is, but getting two slightly different tar.gz files from the same source URL is... bizarre.

**Comment by UdjinM6** [comment]
```
> sha256sum miniupnpc-2.0.20170509.tar.gz
d3c368627f5cdfb66d3ebd64ca39ba54d6ff14a61966dbecb8dd296b7039f16a  miniupnpc-2.0.20170509.tar.gz
> gunzip miniupnpc-2.0.20170509.tar.gz
> sha256sum miniupnpc-2.0.20170509.tar 
b0dbfeafb674b7e2dbb53b9ae071031c34258d94f6224ddb6c17cc20b0f9b3d2  miniupnpc-2.0.20170509.tar
```

**Comment by taw00** [comment]
Oh, wow. Interesting. I am going to inform miniupnpc folks about this. Just wanted to make you aware. It's a non-issue.

**Comment by UdjinM6** [comment]
Yeah, browsers are simply too "smart" - they extract `.gz` files without asking.

**Comment by taw00** [comment]
I talked to the miniupnp guys. They found a header setting that was causing this.
As an aside, they said that they preferred you guys used this server instead:
`https://miniupnp.tuxfamily.org/files/miniupnpc-2.0.20170509.tar.gz`

Reopening for that. Use this information as you see fit. :)

**Comment by nmarley** [comment]
Fixed by #3452
